app/src/combo.c

```diff
@@ -262,37 +260,34 @@ static int clear_candidates() {
 }
 
 static int capture_pressed_key(const zmk_event_t *ev) {
```

Comment:
Would be nice if this took a `const struct zmk_position_state_changed_event*` so it couldn't easily be called with the wrong type of event. Then line 267 could just assign instead of using memcpy as well.

See also my earlier comment about finding a better way to copy events. It could alternatively take a `const struct zmk_position_state_changed*` which is assumed to be part of an event, then use the container/copy function to get the data to copy.

Reply:
Yup, Changed this to do just that.